pkg/action/process.go

```diff
@@ -0,0 +1,101 @@
+package action
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"runtime"
+	"strconv"
+	"strings"
+)
+
+type Process struct {
+	PID  int
+	Path string
+}
+
+// GetAllProcessPaths is an exported function that retrieves the process commands.
+func GetAllProcessPaths() ([]Process, error) {
```

Comment:
It may be overkill for this, but https://github.com/shirou/gopsutil is a well-regarded API for grabbing a process table in Go.

Reply:
I would grealy enjoy a more idiomatic way of doing this 🙂